### PR TITLE
Add settings form JS. Tested with BB 2.0 alpha 1

### PR DESCRIPTION
**Pull request by smarterdigitalltd**

### CODE REVIEW

File: pods-beaver-themer.php [hunk 1]
```diff
@@ -80,6 +80,21 @@ function pods_beaver_admin_nag() {
...
 
 /**
+ * Enqueue assets
+ *
+ * @return void
+ *
+ * @since 1.1
+ */
+function pods_beaver_enqueue_assets() {
+    if ( FLBuilderModel::is_builder_active() ) {
```
Comment (quasel) [inline pods-beaver-themer.php]: should we first check for: wp_doing_ajax() && 
as the settings panel is only loaded from ajax I guess we can't limit it to only posts modules can we?

Comment (smarterdigitalltd) [inline pods-beaver-themer.php]: I don't *think* so... if you check FLBuilder::render_module_settings you will see how BB natively adds the settings JS and CSS for the module (ie settings.js and settings.css), and that is just script tags, not enqueue
screenshot 2017-06-20 16 49 34

**Comment by quasel** [comment]
part of #34

File: assets/js/settings-form.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+(function($) {
+    $(function() {
```
Comment (sc0ttkclark) [inline assets/js/settings-form.js]: Is this second line needed? The first line takes care of load timing already.

**Pull request merged**